base/threadingconstructs.jl

```diff
@@ -65,20 +62,20 @@ function _threadsfor(iter,lbody)
             end
         end
         end
-        # Hack to make nested threaded loops kinda work
-        if threadid() != 1 || in_threaded_loop[]
-            # We are in a nested threaded loop
-            Base.invokelatest(threadsfor_fun, true)
-        else
-            in_threaded_loop[] = true
-            # the ccall is not expected to throw
-            ccall(:jl_threading_run, Cvoid, (Any,), threadsfor_fun)
-            in_threaded_loop[] = false
-        end
+        threading_run(threadsfor_fun)
```

Comment:
Because we are never invoking `threadsfor_fun` with an argument, should the `onethread` argument be removed?